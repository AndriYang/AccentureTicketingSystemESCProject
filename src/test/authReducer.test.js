import authReducer from '../store/reducers/authReducer'

describe('With snapshots', () => {
  it('should return initial state', () =>{
    expect(authReducer(undefined, {})).toMatchSnapshot();
  })

  // it('should handle LOGIN_ERROR', ()=>{
  //   const action = {
  //     type: 'LOGIN_ERROR',
  //   }
  //   expect(ticketReducer(undefined, action)).toMatchSnapshot();
  // })
  //
  // it('should handle LOGIN_SUCCESS', ()=>{
  //   const action = {
  //     type: 'LOGIN_SUCCESS',
  //   }
  //   expect(ticketReducer(undefined, action)).toMatchSnapshot();
  // })
  //
  // it('should handle SIGNOUT_SUCCESS', ()=>{
  //   const action = {
  //     type: 'SIGNOUT_SUCCESS',
  //   }
  //   expect(ticketReducer(undefined, action)).toMatchSnapshot();
  // })
  //
  // it('should handle SIGNUP_SUCCESS', ()=>{
  //   const action = {
  //     type: 'SIGNUP_SUCCESS',
  //   }
  //   expect(ticketReducer(undefined, action)).toMatchSnapshot();
  // })
  //
  // it('should handle CREATE_PROJECT', ()=>{
  //   const action = {
  //     type: 'SIGNUP_ERROR',
  //   }
  //   expect(ticketReducer(undefined, action)).toMatchSnapshot();
  // })
  //
  // it('should handle CREATE_PROJECT', ()=>{
  //   const action = {
  //     type: '',
  //   }
  //   expect(ticketReducer(undefined, action)).toMatchSnapshot();
  // })
})
