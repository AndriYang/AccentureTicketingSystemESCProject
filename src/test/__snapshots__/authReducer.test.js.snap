// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With snapshots should return initial state 1`] = `
Object {
  "authError": null,
}
`;
